app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -118,6 +131,9 @@ protected void onCreate(Bundle savedInstanceState) {
         friendRequestWindow = new Dialog(this);
         friendRequestWindow.setCancelable(false);
 
+
+        isStoragePermissionGranted();
```

Comment:
What does it do? The return value is not used

Reply:
?

Reply:
It basically checks if we are allowed to store stuff onto the users phone. But it is not at the right place here.